Java/Java.sublime-syntax

```diff
@@ -1128,8 +1128,19 @@ contexts:
     - match: \n
       scope: invalid.illegal.newline.java
       pop: true
-    - match: \\.
+    - include: strings-escape
+
+  strings-escape:
+    - match: \\u+\h{4}
+      scope: constant.character.escape.unicode.java
+    - match: \\[0-3][0-7]{2}
+      scope: constant.character.escape.octal.java
+    - match: \\[0-7]{1,2}
+      scope: constant.character.escape.octal.java
```

Comment:
The octal escape rules might be merged to `\\[0-3]?[0-7][0-7]?`

Reply:
Did not notice these 2 can be merged. Thanks for the tip!